examples/menu/src/main.rs

```diff
@@ -0,0 +1,123 @@
+use iced::{
+    executor, Application, Clipboard, Command, Container, Element, Length,
+    Menu, Settings, Text,
+};
+use iced_native::keyboard::{Hotkey, KeyCode, Modifiers};
+
+pub fn main() -> iced::Result {
+    App::run(Settings::default())
+}
+
+#[derive(Debug, Default)]
+struct App {
+    selected: Option<Entry>,
+}
+
+#[derive(Debug, Clone)]
+enum Entry {
+    One,
+    Two,
+    Three,
+    A,
+    B,
+    C,
+}
+
+#[derive(Debug, Clone)]
+enum Message {
+    MenuActivated(Entry),
+}
+
+impl Application for App {
+    type Executor = executor::Default;
+    type Message = Message;
+    type Flags = ();
+
+    fn new(_flags: ()) -> (App, Command<Message>) {
+        (App::default(), Command::none())
+    }
+
+    fn title(&self) -> String {
+        String::from("Menu - Iced")
+    }
+
+    fn menu(&self) -> Menu<Message> {
+        let alt = Modifiers {
+            alt: true,
+            control: false,
+            logo: false,
+            shift: false,
+        };
+        let ctrl_shift = Modifiers {
+            control: true,
+            shift: true,
+            logo: false,
+            alt: false,
+        };
```

Comment:
This is somewhat awkward to work with, maybe we could use something like: `Modifiers::control().and(Modifiers::shift())`?

Reply:
I have changed `Modifiers` to use `bitflags` like `winit` does now in b57d567981bb7ef5f9ff397c210778f211d2de5b.

It should be a bit more ergonomic.